vllm/worker/hpu_model_runner.py

```diff
@@ -2871,7 +2871,7 @@ def try_revert_dummy_output_tokens():
                 htorch.core.mark_step()
                 # Only perform sampling in the driver worker.
                 if not self.is_driver_worker:
-                    continue
+                    return []
```

Comment:
What is the problem of using continue?

Reply:
It leads to the "output not defined" error.